static/src/stylesheets/module/_popup.scss

```diff
@@ -293,3 +293,109 @@ $control-offset: 36 + $gs-gutter/2;
     }
 }
 
+/* Ad feedback menu
+   ========================================================================== */
+.ad-feedback.popup__toggle {
+    float: right;
+
+    // Display the down-pointing arrow to show it's a (currently closed) drop-down menu
+    &:after {
+        content: '';
+        display: inline-block;
+        pointer-events: none;
+        width: 0;
+        height: 0;
+        border-bottom: 0;
+        margin-top: 2px;
+        margin-bottom: 2px;
+        margin-left: 4px;
+        border-left: $popup-toggle-arrow-size solid transparent;
+        border-right: $popup-toggle-arrow-size solid transparent;
+        border-top: $popup-toggle-arrow-size solid #005689;
+    }
+
+    // When active (the menu is open)...
+    &.is-active {
+        &:before {
+            // We don't want to inherit the behaviour of the arrow that points back up to the link that opened it
+            display: none;
+        }
+        // We just want draw the up arrow instead of the down arrow, to show the (currently open) menu can be closed
+        &:after {
+            content: '';
+            display: inline-block;
+            width: 0;
+            height: 0;
+            border-top: 0;
+            margin-top: 2px;
```

Comment:
There are a lot of duplicate styles in this block and the ones above. Maybe you could do this instead:

```
.ad-feedback.popup__toggle {
    float: right;

    &:after {
        pointer-events: none;
        border-bottom: 0;
        border-top: $popup-toggle-arrow-size solid $guardian-brand;
    }

    &:after,
    &.is-active:after {
        content: '';
        display: inline-block;
        width: 0;
        height: 0;
        margin-top: 2px;
        margin-bottom: 2px;
        margin-left: 4px;
        border-left: $popup-toggle-arrow-size solid transparent;
        border-right: $popup-toggle-arrow-size solid transparent;
    }

    &.is-active {
        &:before {
            display: none;
        }
        
        &:after {
            border-top: 0;
            border-bottom: $popup-toggle-arrow-size solid $guardian-brand;
        }
    }
}
```